boot/bootutil/include/bootutil/image.h

```diff
@@ -132,6 +132,8 @@ extern "C" {
                                              * signature
                                              */
 #define IMAGE_TLV_COMP_DEC_SIZE     0x73    /* Compressed decrypted image size */
+#define IMAGE_TLV_UUID_VID          0x80    /* Vendor unique identifier */
+#define IMAGE_TLV_UUID_CID          0x81    /* Device class unique identifier */
```

Comment:
let's use IDs in a group above rather than a new set, we are unfortunately running out of ID groups, use the 0x7x